src/rules/__tests__/no-expect-to-match-object.test.ts

```diff
@@ -0,0 +1,23 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import rule from '../no-expect-to-match-object';
+
+const ruleTester = new TSESLint.RuleTester();
+
+ruleTester.run('no-expect-to-match-object', rule, {
+  valid: [
```

Comment:
Could you chuck in some valid cases for when:

* expect is called without any matcher (i.e `expect(a);`)
* expect is called with a modifier
* expect is called with a modifier + matcher
* `toMatchObject` is called by itself, as if it's a standalone function (i.e `'toMatchObject(...)'`)

Reply:
added tests in https://github.com/jest-community/eslint-plugin-jest/pull/545/commits/0f8ada9303b31b3e7936595529614921d90c64f4